yat-docs/docs/partner-guides/1_crypto_wallets.md

```diff
@@ -0,0 +1,304 @@
+- `0x1003` Bitcoin address
+- `0x1004` Ethereum address
+- `0x3fff` Generic crypto address
+
+Here's an addresses parameter value that defines BTC and ETH addresses before percent-encoding:
+```
+0x1003=1NDyJtNTjmwk5xPNhjgAMu4HDHigtobu1s|0x1004=108dEFa0272dC118EF03a7993e4fC7A8AcF3a3d1```
+```
+
+Here's the same parameter value after percent-encoding:
+```
+0x1003%3D1NDyJtNTjmwk5xPNhjgAMu4HDHigtobu1s%7C0x1004%3D108dEFa0272dC118EF03a7993e4fC7A8AcF3a3d1
+```
+
+And here's an imaginary URL for a partner app with the path parameter (e.g. coolwalletapp) that has the refresh_token, eid and the percent-encoded addresses query parameters:
+
+```
+https://y.at/partner/coolwalletapp?eid=🗞️💉⛸️🆓🔮&addresses=0x1003%3D1NDyJtNTjmwk5xPNhjgAMu4HDHigtobu1s%7C0x1004%3D108dEFa0272dC118EF03a7993e4fC7A8AcF3a3d1&refresh_token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJiNmI2ZGE4MS0yNTBlLTQ1NmUtYTYxYS01ZjQyNmFlOTJhMjIiLCJpc3MiOiJlbW9qaS1pZC10b2tlbi1pc3N1ZXIiLCJleHAiOjE2MjAzNzczMjgsInNjb3BlcyI6WyJjYXJ0OnNob3ciLCJjYXJ0OnVwZGF0ZSIsInVzZXI6Y3JlYXRlQXBpS2V5IiwiZWRpdGlvbjpyZWFkIiwiZW1vamk6OnRyYW5zZmVyIiwibG9vdGJveDp1c2UiLCJvcmRlcjpyZWFkU2VsZiIsIm9yZ2FuaXphdGlvbkxpc3Q6cmVhZCIsInBheW1lbnRNZXRob2Q6ZGVzdHJveSIsInBheW1lbnRNZXRob2Q6cmVhZCIsInBheW1lbnRNZXRob2Q6c2V0RGVmYXVsdCIsInVzZXI6ZGVsZXRlU2VsZiIsInVzZXJJbnRlcmVzdDpkZWxldGUiLCJ1c2VySW50ZXJlc3Q6cmVhZCIsInVzZXJJbnRlcmVzdDp3cml0ZSIsInVzZXI6d3JpdGVTZWxmIl0sImlzc3VlZCI6MTYyMDM3NjQyOCwiYWN0aXZlMmZhIjowfQ.FZBzoe8Zky9Pl7WCNXwg5KphVO4FNTKxJKX87w9sHW0
+```
+
+
+### Redirection of the User from Yat Web Flow to the Partner Application
+
+The user can choose to purchase a new Yat on the Yat web application. After the successful
+completion of this flow, the Yat web application calls the return URL previously defined by the partner application development team.
+
+**Return URL:** To be agreed upon by the Yat and partner development teams in pre-integration setup.
+
+** Query Parameters: **
+
+`eid:` The Yat that was purchased, if any.
+
+`refresh_token:` The same refresh token that was sent to the web application by the partner application in [Redirection of the User to the Yat Web Application](#redirection-of-the-user-to-the-yat-web-application).
+
+The partner application receives the query parameters as a result of the redirection from the Yat web application to the partner application, and the integration flow is completed.
+
+## 2. Connect existing Yat(s)
+
+Users who already own one or more Yats can connect their Yat(s) with their wallet. We recommend that partner apps enable the user to enter the journey from:
+
+* Push notification announcing the partnership to all users
+* Splash screens on app launch
+* Menu item within the app’s Settings or Profile section
+
+### 2A. Connect existing Yat(s) from Onboarding
+
+<a href="/img/flow2a-lrg.png" target="_blank"><img src="/img/flow2a-sml.png"/></a>
+
+### 2B. Connect existing Yat(s) from Settings
+
+<a href="/img/flow2b-lrg.png" target="_blank"><img src="/img/flow2b-sml.png"/></a>
+
```

Comment:
In reference to @shanimal08's point [here](https://github.com/yat-labs/yat-docs/pull/15/commits/e07a67c6e243c133922bcaa9709a2296ef06bbe1#r712977825): adding the following. @NovaT82  can you add this text? 

The partner application user needs to get redirected to the Yat website to Connect a Yat (Flow [2](#2-connect-existing-yats)). A number of query parameters are necessary for the Yat web application to correctly identify the user and link any necessary data to the user's Yat.

Web App URL: `https://y.at/partner/{partner_path}/link-email`

Reply:
@shanimal08 which query params apply for this if at all?

Reply:
hai @djangolin yes please! it would be ideal if we could get the `refresh_token`, `eid`, and `addresses` 

though the cases should all be handled now if one or more are missing😅

Reply:
@shanimal08 I've done these edits should turn up once @NovaT82 accepts them. :)